manim/mobject/coordinate_systems.py

```diff
@@ -59,6 +59,44 @@
 class CoordinateSystem:
     """
     Abstract class for Axes and NumberPlane
+
+    Examples
+    --------
+
+    .. manim:: CoordSysExample
+        :save_last_frame:
+
+        class CoordSysExample(Scene):
+            def construct(self):
+                # When drawing Axes, the location of the ticks
+                # on the axes depend on x_range and y_range.
+                grid = Axes(
+                    x_range=[0, 1, 0.05],
+                    y_range=[0, 1, 0.05],
+                    x_length=9,
+                    y_length=5.5,
+                    axis_config={"decimal_number_config": {"num_decimal_places": 2}},
+                    x_axis_config={"numbers_to_include": np.arange(0, 1 + 0.1, 0.1), "number_scale_value": 0.5},
+                    y_axis_config={"numbers_to_include": np.arange(0, 1 + 0.1, 0.1), "number_scale_value": 0.5},
+                    tips=False,
+                )
+
+                graphs = VGroup()
+                # Label for x-axis and y-axis.
+                grid_labels = grid.get_axis_labels(x_label="x", y_label="y")
+
+                for n in np.arange(0.05, 1 + 0.05, 0.05):
+                    graphs += grid.get_graph(lambda x: x ** (1 / n), x_range=[0, 1], color=WHITE)
+                    graphs += grid.get_graph(lambda x: x ** (n), x_range=[0, 1], color=WHITE, use_smoothing=False)
+
+                # Extra lines and labels for point (1,1)
+                graphs += grid.get_line_from_axis_to_point(0, grid.c2p(1, 1, 0), color=BLUE)
+                graphs += grid.get_line_from_axis_to_point(1, grid.c2p(1, 1, 0), color=BLUE)
```

Comment:
In my opinion, `get_line_from_axis_to_point` should be considered more of an internal function, so I'd rather `get_vertical_line` and `get_horizontal_line` be used here.

Reply:
My last commit should have fixed this issue.